index.html

```diff
@@ -306,12 +304,14 @@ <h2><code>PointerEvent</code> Interface</h2>
     readonly        attribute long        tiltX;
     readonly        attribute long        tiltY;
     readonly        attribute long        twist;
-    readonly        attribute double      altitudeAngle;
-    readonly        attribute double      azimuthAngle;
     readonly        attribute DOMString   pointerType;
     readonly        attribute boolean     isPrimary;
     [SecureContext] sequence&lt;PointerEvent> getCoalescedEvents();
     sequence&lt;PointerEvent> getPredictedEvents();
+    static                    double      getAzimuthAngle(long tiltX, long tiltY);
```

Comment:
Alternatively, we can just add getAzimuth and getAltitude functions with no parameter. It gives away the transformation from the current data of the pointerevents.
It does have one caveat. If a test program wants to generate a synthetic pointerevent with a given azimuth/altitude it has to do it by hand and pass it to the constructor. WDYT?

Reply:
> Alternatively, we can just add getAzimuth and getAltitude functions with no parameter. It gives away the transformation from the current data of the pointerevents.

Having `getAzimuthAngle` and `getAltitudeAngle` methods on the PointerEvent makes sense.

> It does have one caveat. If a test program wants to generate a synthetic pointerevent with a given azimuth/altitude it has to do it by hand and pass it to the constructor.

This is the use-case for which I added `static long getTiltX(double azimuth, double altitude)` and `static long getTiltY(double azimuth, double altitude)` methods: to allow easy `tiltX`/`tiltY` calculation given `azimuth`/`altitude` pair.